data/tokens.json

```diff
@@ -1258,6 +1258,10 @@
         "value": "1px",
         "type": "spacing"
       },
+      "-010": {
```

Comment:
I think it might be best if the negative figure variants appear before or after the positive variants in terms of findability. What do you reckon?